src/main/java/com/wavedroid/musicbrainz/dao/MusicbrainzDao.java

```diff
@@ -176,6 +177,61 @@
             "         AS tbl) AS tbl2\n" +
             "ORDER BY artist, rg_year, rg_month\n";
 
+    private static final String RELEASE_BY_MBID = "SELECT\n" +
```

Comment:
I'd prefer to see a few lines added, not the whole SQL being replaced

Reply:
okay, I'll provide changes